### ISSUE TITLE
Can't find environmental variable

### ISSUE THREAD

**Issue by wcjord** [google/google-auth-library-nodejs#214]
I keep getting:
Error: Could not load the default credentials. Browse to https://developers.google.com/accounts/docs/application-default-credentials for more information.
  at <path to project>\chat1\node_modules\google-auth-library\lib\auth\googleauth.js:316:21

There are two ways of manually fixing it but they aren't viable long-term. I can go into the google-auth-library package and hard-code the path like this.
//var credentialsPath = that._getEnv('GOOGLE_APPLICATION_CREDENTIALS');
var credentialsPath = <path to file>; 

I can also go into my computer's system settings to set the variable.

The normal and preferred methods of setting environmental variables aren't working. I've tried process.env.GOOGLE_APPLICATION_CREDENTIALS=<path to file> in a file that runs at the beginning of the project. I've also done process.env.['GOOGLE_APPLICATION_CREDENTIALS']='path to file' to be safe. I've also tried using dotenv with the .env file. 

I imagine this is some problem with scoping? I'm at a loss...

**Comment by JustinBeckwith** [comment]
This error happens when all of our approaches to loading the credentials fail.  We check for:
- The `GOOGLE_APPLICATION_CREDENTIALS` env var (the path to the file)
- A logged in user with the Cloud SDK
- Metadata service on GCE

In the samples here, I verified that this works for me:
https://github.com/google/google-auth-library-nodejs/blob/next/examples/adc.js

`$ GOOGLE_APPLICATION_CREDENTIALS=jwt.keys.json node adc.js`

Can you share the exact code you're using?  And exactly how you're trying to set the `GOOGLE_APPLICATION_CREDENTIALS`?

Note:  this is using the 1.0 alpha, `google-auth-library@1.0.0-alpha.1`.

**Comment by wcjord** [comment]
@JustinBeckwith Thank for the response! I found the problem!

I didn't realize that the authentication was done as soon as the package was required because the error message doesn't print until you try to use a cloud service. I was requiring the package, then setting the environmental variable, then calling the API. I found out what was happening when I set a console.log in the actually getEnv function in google-auth-library and realized it was being called before setting GOOGLE_APPLICATION_CREDENTIALS. 

Thank you for the response!
